python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -18,15 +18,21 @@
 SLACK_WEBHOOK_CONN = os.getenv("SLACK_WEBHOOK_CONN", "http_slack")
 SLACK_USERNAME = os.getenv("SLACK_USERNAME", "airflow_app")
 
+is_runtime_release = Variable.get("IS_RUNTIME_RELEASE", default_var="False")
```

Comment:
Could this be an Airflow configuration as opposed to a `Variable`?

The advantage of using config is that in line 32, we'd be able to use `getboolean`:
https://github.com/apache/airflow/blob/54f1fb0574a6ecf8f415bbf6da1aaf6f1999bb29/airflow/configuration.py#L765

Reply:
Not able to add a new config option however converted to bool value after taking srt value from env